### ISSUE TITLE
has no exported member 'Unit'

### ISSUE THREAD

**Issue by SurajEmorphis** [dmtrKovalenko/date-io#460]
ERROR in node_modules/@date-io/date-fns/build/date-fns-utils.d.ts:1:33 - error TS2305: Module '"../../core/IUtils"' has no exported member 'Unit'.

node version- 12.18.3
npm version- 6.14.6
angular-cli version- 10.1.1

**Comment by dmtrKovalenko** [comment]
What version are you using?

**Comment by zapo** [comment]
I have the same issue
```
@date-io/date-fns@2.9.1
@date-io/core@2.9.0
```

**Comment by zapo** [comment]
Not sure if this is the proper way but I'm working around it with adding this to an included d.ts file:
```
import { IUtils, DateIOFormats } from "@date-io/core/IUtils";
declare module "@date-io/core/IUtils" {
  export type Unit = string;
}
```

**Comment by madrus** [comment]
Having same issue. Noticed that IUtils and DateIOFormats are exported interfaces while Unit is an exported type. Not sure if this is a relevant observation. Just thinking aloud.

**Comment by SurajEmorphis** [comment]
> What version are you using?

"@date-io/date-fns": "^2.6.1",

**Comment by madrus** [comment]
> Not sure if this is the proper way but I'm working around it with adding this to an included d.ts file:
> 
> ```
> import { IUtils, DateIOFormats } from "@date-io/core/IUtils";
> declare module "@date-io/core/IUtils" {
>   export type Unit = string;
> }
> ```

worked for me when I added this code to src/global.d.ts file in my project.

**Comment by wingsbob** [comment]
Just to add clarity, the code is there on master: https://github.com/dmtrKovalenko/date-io/blob/master/packages/core/IUtils.d.ts#L62-L71
But the tag for 2.9.0 doesn't include it: https://github.com/dmtrKovalenko/date-io/blob/v2.9.0/packages/core/IUtils.d.ts

**Comment by dmtrKovalenko** [comment]
Published all packages `v2.10.0` with the fixes.

https://unpkg.com/@date-io/core@2.10.0/IUtils.d.ts

**Comment by SurajEmorphis** [comment]
Thank you

> Published all packages `v2.10.0` with the fixes.
> 
> https://unpkg.com/@date-io/core@2.10.0/IUtils.d.ts

Thank you
